src/base/search/searchdownloadhandler.cpp

```diff
@@ -41,7 +41,7 @@ SearchDownloadHandler::SearchDownloadHandler(const QString &pluginName, const QS
     , m_manager {manager}
     , m_downloadProcess {new QProcess(this)}
 {
-    m_downloadProcess->setEnvironment(QProcess::systemEnvironment());
+    m_downloadProcess->setProcessEnvironment(manager->proxyEnvironment());
```

Comment:
Wouldn't it be better to use `manager` parameter only to initialize `m_manager` member variable and then use such a member?
```suggestion
    m_downloadProcess->setProcessEnvironment(m_manager->proxyEnvironment());
```